packages/hoprd/Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 # -- BASE STAGE --------------------------------
 
-FROM node:12.9.1-buster AS base
+FROM node@sha256:966f569423c977e862b4e5b1f01234720dd212394cd13f5904d47864aece5417 AS base
```

Comment:
Can we at least comment where this hash comes from? 
This is pretty unmaintainable as-is.

Reply:
Ah, definitely, good call. Adding comment.

Reply:
Hi @peterbraden, it's me, Liran, from the tweet mentioned in the comment :-)
The SHA is taken from the image digest itself. You can validate it on the Docker Hub page, or by pulling the relevant image tag alias for it.

I actually suggest that you apply the following fix:
```suggestion
FROM node:12.9.1-buster@sha256:966f569423c977e862b4e5b1f01234720dd212394cd13f5904d47864aece5417 AS base
```

^ this above now is both easily maintainable, as you know exactly what base image tag you're using (`node:12.9.1-buster`) and also you are requiring a specific image digest build for it, to have deterministic builds.

Reply:
Thanks for dropping by @lirantal! Updating your comment in a new PR